chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -1,709 +1,806 @@
 Blvd.\",\"line2\":\"Suite 200\",\"city\":\"Mechanicsburg\",\"state\":\"PA\",\"zipcode\":\"17050\",\"country\":\"USA\"},\"retired\":false,\"retirementDate\":null}},{\"id\":6002,\"chplProductNumber\":\"CHP-024149\",\"lastModifiedDate\":1593499211217,\"edition\":\"2014\",\"certificationDate\":1412553600000,\"certificationStatus\":\"Retired\",\"curesUpdate\":null,\"acb\":{\"id\":6,\"acbCode\":\"07\",\"name\":\"ICSA Labs\",\"website\":\"https://www.icsalabs.com\",\"address\":{\"addressId\":7,\"line1\":\"1000 Bent Creek Blvd.\",\"line2\":\"Suite 200\",\"city\":\"Mechanicsburg\",\"state\":\"PA\",\"zipcode\":\"17050\",\"country\":\"USA\"},\"retired\":false,\"retirementDate\":null}},{\"id\":6310,\"chplProductNumber\":\"CHP-023174\",\"lastModifiedDate\":1593493657165,\"edition\":\"2014\",\"certificationDate\":1403827200000,\"certificationStatus\":\"Retired\",\"curesUpdate\":null,\"acb\":{\"id\":6,\"acbCode\":\"07\",\"name\":\"ICSA Labs\",\"website\":\"https://www.icsalabs.com\",\"address\":{\"addressId\":7,\"line1\":\"1000 Bent Creek Blvd.\",\"line2\":\"Suite 200\",\"city\":\"Mechanicsburg\",\"state\":\"PA\",\"zipcode\":\"17050\",\"country\":\"USA\"},\"retired\":false,\"retirementDate\":null}}]},{\"versionId\":7458,\"version\":\"8\",\"details\":null,\"lastModifiedDate\":\"1569517081783\",\"listings\":[{\"id\":9643,\"chplProductNumber\":\"14.07.07.2901.FD01.01.01.1.180910\",\"lastModifiedDate\":1593482180903,\"edition\":\"2014\",\"certificationDate\":1536537600000,\"certificationStatus\":\"Retired\",\"curesUpdate\":null,\"acb\":{\"id\":6,\"acbCode\":\"07\",\"name\":\"ICSA Labs\",\"website\":\"https://www.icsalabs.com\",\"address\":{\"addressId\":7,\"line1\":\"1000 Bent Creek Blvd.\",\"line2\":\"Suite 200\",\"city\":\"Mechanicsburg\",\"state\":\"PA\",\"zipcode\":\"17050\",\"country\":\"USA\"},\"retired\":false,\"retirementDate\":null}},{\"id\":9642,\"chplProductNumber\":\"14.07.07.2901.FD01.01.01.1.180909\",\"lastModifiedDate\":1593528534234,\"edition\":\"2014\",\"certificationDate\":1536451200000,\"certificationStatus\":\"Retired\",\"curesUpdate\":null,\"acb\":{\"id\":6,\"acbCode\":\"07\",\"name\":\"ICSA Labs\",\"website\":\"https://www.icsalabs.com\",\"address\":{\"addressId\":7,\"line1\":\"1000 Bent Creek Blvd.\",\"line2\":\"Suite 200\",\"city\":\"Mechanicsburg\",\"state\":\"PA\",\"zipcode\":\"17050\",\"country\":\"USA\"},\"retired\":false,\"retirementDate\":null}},{\"id\":9972,\"chplProductNumber\":\"15.04.04.2901.FDB1.08.00.1.190411\",\"lastModifiedDate\":1584042323020,\"edition\":\"2015\",\"certificationDate\":1554940800000,\"certificationStatus\":\"Withdrawn by Developer\",\"curesUpdate\":false,\"acb\":{\"id\":3,\"acbCode\":\"04\",\"name\":\"Drummond Group\",\"website\":\"http://www.drummondgroup.com\",\"address\":{\"addressId\":2,\"line1\":\"13359 North Hwy. 183\",\"line2\":\"Suite B 406-238\",\"city\":\"Austin\",\"state\":\"Texas\",\"zipcode\":\"78750\",\"country\":\"USA\"},\"retired\":false,\"retirementDate\":null}}]}]}]},\"developerIds\":[1902]}"
```

Comment:
Isn't this developer related information sensitive? Or this developer information is public?

Reply:
this developer information is public, GET /developers or /developers/{developerId} returns all this info. It was discussed with the team prior to adding the request body on this test